jetty-core/jetty-websocket/jetty-websocket-core-common/src/main/java/org/eclipse/jetty/websocket/core/WebSocketConnection.java

```diff
@@ -61,7 +61,7 @@ public class WebSocketConnection extends AbstractConnection implements Connectio
     private final WebSocketCoreSession coreSession;
     private final Flusher flusher;
     private final Random random;
-    private long demand;
+    private Boolean demand = false;
```

Comment:
I know I often use a Boolean to create a tri-state.... but it is often a mistake.   Would an enum be clearer here?   If you keep the Boolean, then at least add a comment what the null value means.

Reply:
done